valhalla/baldr/traffictile.h

```diff
@@ -52,12 +52,13 @@ struct TrafficSpeed {
   uint64_t speed1 : 7;        // 0-255kph in 2kph resolution (access with `get_speed(0)`)
   uint64_t speed2 : 7;
   uint64_t speed3 : 7;
-  uint64_t breakpoint1 : 8; // position = length * breakpoint1 * 255
-  uint64_t breakpoint2 : 8; // position = length * breakpoint2 * 255
-  uint64_t congestion1 : 6; // Stores 0 (unknown), or 1->63 (no congestion->max congestion)
-  uint64_t congestion2 : 6; //
-  uint64_t congestion3 : 6; //
-  uint64_t spare : 2;
+  uint64_t breakpoint1 : 8;   // position = length * (breakpoint1 / 255)
+  uint64_t breakpoint2 : 8;   // position = length * (breakpoint2 / 255)
+  uint64_t congestion1 : 6;   // Stores 0 (unknown), or 1->63 (no congestion->max congestion)
+  uint64_t congestion2 : 6;   //
+  uint64_t congestion3 : 6;   //
+  uint64_t has_incidents : 1; // Are there incidents on this edge in the corresponding incident tile
```

Comment:
tells the graph reader that it should consider fetching incidents for this edge. due to synchronization it still may be possible that this disagrees with what will be in the incident tiles but it shouldnt matter. if it says they are there and they are not the worst case is we looked and found none in the graph reader function above. no big deal